src/IssueViz/Editor/LocationNavigator.cs

```diff
@@ -47,33 +49,43 @@ internal LocationNavigator(IDocumentNavigator documentNavigator, IIssueSpanCalcu
             this.logger = logger;
         }
 
-        public bool TryNavigate(IAnalysisIssueLocation location)
+        public bool TryNavigate(IAnalysisIssueLocationVisualization locationVisualization)
```

Comment:
It now accepts LocationViz for 2 reasons:
1. To use `locationVisualization.CurrentFilePath` in-case of renames.
2. To update `locationVisualization.Span` with the calculated value.